src/server/db/schema.ts

```diff
@@ -180,6 +180,7 @@ export const reviews = createTable(
     })
       .defaultNow()
       .notNull(),
+    completed: boolean("completed").default(false),
```

Comment:
This is great! Thanks for adding this column, I forgot that we would need it LOL. Could you run `npm run db:generate` and npm run db:migrate` to add db migration scripts for adding this column?

Reply:
before you run these, can you make `referral` default to false?

Reply:
Changed the referral default but I got this message when running migrate. Is there anything I gotta do?
![image](https://github.com/hackwestern/hackwestern/assets/58274342/2d64fac3-dac2-4593-8e65-74fc297e9819)

Reply:
nope! those are fine, I'm not sure why they always show up